src/AppKit/Enums.cs

```diff
@@ -2112,6 +2122,7 @@ public static class NSFileTypeForHFSTypeCode {
 	
 	// These constants specify the possible states of a drawer.
 	[Native]
+	[Deprecated (PlatformName.MacOSX, 10, 13, message: "Drawers are deprecated; consider using 'NSSplitViewController'")]
```

Comment:
Yup exactly: https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#rule-4

Here you can just say: `Use 'NSSplitViewController' instead.`

The IDE will show the user the following tooltip:

```
'NSDrawerState' was deprecated in macOS 10.13.
Use 'NSSplitViewController' instead.
```